lesson_06/homework_06.py

```diff
@@ -0,0 +1,76 @@
+# separated line
+def sep_line():
+    print('-' * 100)
+
+
+# homework 6.1
+def homework_6_1():
+    sep_line()
+
+    input_string = input('Enter anything: ')
+    set_string = set(input_string)
+
+    count_set_string = 0
+    for _ in set_string:
+        count_set_string += 1
+
+    if count_set_string > 10:
+        print('Result: True')
+    else:
+        print('Result: False')
```

Comment:
Prints could be more informative.
When I ran your program, I couldn't understand from output what is going on.